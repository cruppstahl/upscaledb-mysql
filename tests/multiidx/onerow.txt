DROP TABLE IF EXISTS test;
CREATE TABLE test (value INTEGER NOT NULL, INDEX(value), num INTEGER PRIMARY KEY) ENGINE=upscaledb;
INSERT INTO test VALUES(1, 1);
SELECT * FROM test;
UPDATE test SET value = value + 1 WHERE num <= 4;
SELECT * FROM test;

