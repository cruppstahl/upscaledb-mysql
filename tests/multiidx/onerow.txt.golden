--------------
DROP TABLE IF EXISTS test
--------------

--------------
CREATE TABLE test (value INTEGER NOT NULL, INDEX(value), num INTEGER PRIMARY KEY) ENGINE=upscaledb
--------------

--------------
INSERT INTO test VALUES(1, 1)
--------------

--------------
SELECT * FROM test
--------------

value	num
1	1
--------------
UPDATE test SET value = value + 1 WHERE num <= 4
--------------

--------------
SELECT * FROM test
--------------

value	num
2	1
